README.md

```diff
@@ -1,31 +1,55 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
+![](https://wakatime.com/badge/user/be79098d-c59e-407c-8952-0f274bb9e265/project/c8c5912d-d823-44a6-9418-52768d49728b.svg)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Covid Tracker
+
+> A web application which designed to automatically alert people to whether they are at high risk of having the virus.
+## Technologies used
+
+- React
+- HTML5
+- CSS3
+- Jest
+- JavaScript
+- Jest
+
+## Live Demo
+
+[Live Demo Link](https://miftah1991.github.io/react-covid-tracker/)
```

Comment:
- [ ] Currently, your live demo is not working properly. Instead of showing the app, it shows the readme file. kindly fix this.

:+1: :smile: